tests/models/TestBggCompanionVariables.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
+from collections import OrderedDict
+
+from src.models.BoardGame import BoardGame
+
+
+class TestGetCollectionVariables:
```

Comment:
Maybe rename this to `TestGetCollectionVariables` to `TestGetCollectionData`

Reply:
Definitely makes sense, will do!